src/ia/js/IAIndex.js

```diff
@@ -271,6 +270,8 @@
             var selector_all;
             var text_all;
             var tot_count = 0;
+
+            $("#ia_index_header h2").text("Showing " + $(".ia-item:visible").length + " Instant Answers");
```

Comment:
why not give an id for the h2 to address it directly? seems superfluous to search for it within the #ia_index_header.